pulp-in-one-container.md

```diff
@@ -23,9 +23,15 @@ $ mkdir settings pulp_storage pgsql containers
 $ echo "CONTENT_ORIGIN='http://$(hostname):8080'
 ANSIBLE_API_HOSTNAME='http://$(hostname):8080'
 ANSIBLE_CONTENT_HOSTNAME='http://$(hostname):8080/pulp/content'
+TOKEN_SERVER = "http://(hostname):24817/token/"
```

Comment:
The port is 8080 in this case. Also, I am not sure this will work since the user would need to run the openssl utility to create the keys.